engine/src/flutter/shell/platform/embedder/embedder_semantics_update.cc

```diff
@@ -279,6 +321,7 @@ void EmbedderSemanticsUpdate2::AddNode(const SemanticsNode& node) {
       increased_value_attributes.attributes,
       decreased_value_attributes.count,
       decreased_value_attributes.attributes,
+      &flags_.back(),
```

Comment:
GitHub won't let me comment in places where there isn't a diff, but I'd be tempted to add a comment to the `SemanticsNodeToInt()` function above -- similar to the one on the `flags` field itself, that states that it's only for backward compatibility and that it only contains a subset of the flags.